js/main.js

```diff
@@ -0,0 +1,113 @@
+const navigation = document.querySelector('.mobile-nav');
+const expandMenu = () => {
+  navigation.classList.remove('animate-left');
+  navigation.classList.add('animate-right');
+};
+
+const closeMenu = () => {
+  navigation.classList.remove('animate-right');
+  navigation.classList.add('animate-left');
+};
+
+document.querySelector('.open').addEventListener('click', expandMenu);
+
+const closeClass = document.querySelectorAll('.close');
+closeClass.forEach(((element) => element.addEventListener('click', closeMenu)));
+
+const header = document.querySelector('.page-header');
+
+document.addEventListener('scroll', () => {
+  if (window.scrollY > 0) {
+    if (!header.classList.contains('white-bg')) {
+      header.classList.add('white-bg');
+    }
+  } else {
+    header.classList.remove('white-bg');
+  }
+});
+
+const supportList = [
+  {
+    name: 'Alex Abewa',
+    postion: 'CTO at DATStar Technology',
+    imagePath: './images/avator.png',
+    description: 'Alex is one of a dedicated support team who solves any issue related to GSP and speedlimeter. He has a long time experice in this domain.',
+  },
+  {
+    name: 'Alex Abewa',
+    postion: 'CTO at DATStar Technology',
+    imagePath: './images/avator.png',
+    description: 'Alex is one of a dedicated support team who solves any issue related to GSP and speedlimeter. He has a long time experice in this domain.',
+  },
+  {
+    name: 'Alex Abewa',
+    postion: 'CTO at DATStar Technology',
+    imagePath: './images/avator.png',
+    description: 'Alex is one of a dedicated support team who solves any issue related to GSP and speedlimeter. He has a long time experice in this domain.',
+  },
+  {
+    name: 'Daniel Getachew',
+    postion: 'CTO at DATStar Technology',
+    imagePath: './images/avator.png',
+    description: 'Alex is one of a dedicated support team who solves any issue related to GSP and speedlimeter. He has a long time experice in this domain.',
+  },
+  {
+    name: 'Daniel Getachew',
+    postion: 'CTO at DATStar Technology',
+    imagePath: './images/avator.png',
+    description: 'Alex is one of a dedicated support team who solves any issue related to GSP and speedlimeter. He has a long time experice in this domain.',
+  },
+  {
+    name: 'Daniel Getachew',
+    postion: 'CTO at DATStar Technology',
+    imagePath: './images/avator.png',
+    description: 'Alex is one of a dedicated support team who solves any issue related to GSP and speedlimeter. He has a long time experice in this domain.',
+  },
+];
+
```

Comment:
- [ ] [OPTIONAL] I loved your website design 👍. However to make it better, please add real human faces instead of one avatar in 6 cards. I am sure you can do it easily. Here are some websites to get free high-resolution images like [Unsplash](https://unsplash.com/) and [Pexels](https://www.pexels.com/).